### PR TITLE
Preparation for 0.2.1 release

### PR DESCRIPTION
**Pull request by tshoang**
- Create Tests plug-in and features, include them in SDK
- Inclusion generator plug-in: Use adaptation of EMF Translator for Event-B.

### CODE REVIEW

File: ac.soton.eventb.emf.inclusion.feature/feature.xml [hunk 1]
```diff
@@ -44,6 +43,8 @@
       <import plugin="org.rodinp.core" version="1.7.0" match="greaterOrEqual"/>
       <import plugin="ac.soton.eventb.emf.inclusion" version="0.1.0" match="greaterOrEqual"/>
       <import plugin="org.eclipse.emf.transaction" version="1.8.0" match="greaterOrEqual"/>
+      <import plugin="org.eventb.emf.core" version="5.0.0" match="compatible"/>
+      <import plugin="ac.soton.emf.translator.eventb" version="0.0.1" match="greaterOrEqual"/>
    </requires>
```
Comment (cfsnook) [inline ac.soton.eventb.emf.inclusion.feature/feature.xml]: make all dependencies 'compatible'

File: ac.soton.eventb.emf.inclusion.tests.feature/build.properties [hunk 2]
```diff
@@ -0,0 +1,8 @@
+bin.includes = feature.xml,\
+               feature.properties,\
+               build.properties,\
+               .project
+src.includes = feature.xml,\
+               feature.properties,\
+               build.properties,\
+               .project
```
Comment (cfsnook) [inline ac.soton.eventb.emf.inclusion.tests.feature/build.properties]: Does bin need build.properties and .project?

File: ac.soton.eventb.emf.inclusion.generator/src/ac/soton/eventb/emf/inclusion/generator/InclusionAdaptor.java [hunk 3]
```diff
@@ -48,7 +49,7 @@
  *
  */
 
-public class InclusionAdaptor  implements IAdapter {
+public class InclusionAdaptor extends EventBTranslatorAdapter implements IAdapter {
```
Comment (cfsnook) [inline ac.soton.eventb.emf.inclusion.generator/src/ac/soton/eventb/emf/inclusion/generator/InclusionAdaptor.java]: Why override methods without changing? The ones in EventBTranslatorAdapter are more up to date. Maybe declare EventBTranslatorAdapter directly and delete InclusionAdapter.

Comment (tshoang) [reply ac.soton.eventb.emf.inclusion.generator/src/ac/soton/eventb/emf/inclusion/generator/InclusionAdaptor.java]: Because we copied it initially ...

**Comment by tshoang** [comment]
Changes have been made according to @cfsnook's comments

**Pull request merged**